abci: remove counter app (#6684)
tendermint/tendermint: docs/app-dev/getting-started.md

```

> Now you should have the `abci-cli` installed; you'll see a couple of

Reviewer: ```suggestion
Now you should have the `abci-cli` installed; you'll notice the `kvstore`
```

> commands (`kvstore`) that are example applications written

Reviewer: ```suggestion
command, an example application written
```
